sdk/core/Azure.Core/src/Pipeline/DisposableHttpPipeline.cs

```diff
@@ -6,10 +6,15 @@
 namespace Azure.Core.Pipeline
 {
     /// <summary>
-    /// A wrapper type for <see cref="HttpPipeline"/> that exposes a <see cref="Dispose"/> method for disposal of the underlying <see cref="HttpPipelineTransport"/>.
+    /// A wrapper type for <see cref="HttpPipeline"/> that exposes a Dispose method for disposal of the underlying <see cref="HttpPipelineTransport"/>.
```

Comment:
Not a wrapper type anymore. "HttpPipeline implementation that might contain resources that need to be disposed after use" ?

Reply:
I wouldn't focus on HttpTransport detail as much. Just say "you need to dispose this"